target/nanos2/script.ld

```diff
@@ -168,6 +171,18 @@ SECTIONS
   .debug_funcnames 0 : { *(.debug_funcnames) }
   .debug_typenames 0 : { *(.debug_typenames) }
   .debug_varnames  0 : { *(.debug_varnames) }
+
+  .ARM.attributes : { *(.ARM.attributes) }
```

Comment:
Not sure this is necessary ... In my PR I added it to try and force the metadata to be at the end of the file (I think), but is it really important ?